h2o-docs/src/product/welcome.rst

```diff
@@ -460,6 +460,11 @@ Supported Versions
 -  ``-mapperXmx``, ``-nodes``, and ``-output`` are required
 -  Root permissions are not required - just unzip the H2O .zip file on any single node
 
+Java Requirements
```

Comment:
We already have one title with java requirements. Can we adjust the current one on line 34? IMHO it would be better